scipy/stats/tests/test_multivariate.py

```diff
@@ -2000,6 +2000,16 @@ def test_frozen_distribution(self):
         assert_equal(rvs1, rvs2)
         assert_equal(rvs1, rvs3)
 
+    def test_uniform_circle(self):
+        # test that for uniform 2D samples on the circle the resulting
+        # angles follow a uniform distribution
+        circular_dist = random_direction(2)
+        samples = circular_dist.rvs(size=1000, random_state=158023532469097)
+        angles = np.arctan2(samples[:, 1], samples[:, 0])
+        # normalize angles to range [0, 1]
+        angles /= 2*np.pi
+        kstest_result = kstest(angles, uniform.cdf)
+        assert kstest_result.pvalue > 0.05
```

Comment:
Is this what you had in mind @mdhaber ?

Reply:
Yes, although you can also do the same thing in higher dimensions, and you can project onto a plane oriented in any direction (if you wanted to). It is a weak test for detecting whether something is obviously wrong rather than confirm that it's right, but that's what I was suggesting.